### ISSUE TITLE
Missing Item number and warehouse 

### ISSUE THREAD

**Issue by IamUser1337** [MicrosoftLearning/MB-335-Dynamics-365-Supply-Chain-Management-Expert#4]
[M4 Configure production control_masterplanning.md](https://github.com/MicrosoftLearning/MB-335-Dynamics-365-Supply-Chain-Management-Expert/blob/master/Instructions/Labs/M4%20Configure%20production%20control_masterplanning.md)

#### Step: 02 & 06

Description of issue

Had to use L0001 instead of L0025

and warehouse 13 instead of 11 

I did not check the other companies in the top left, maybe we were supposed to switch at some point? 

![image](https://github.com/MicrosoftLearning/MB-335-Dynamics-365-Supply-Chain-Management-Expert/assets/72228536/b4322a4f-81b6-4a4e-83c1-96c2f148a7d5)

**Comment by mstall1ngs** [comment]
Thanks for reporting this on GitHub! Module 4 / Case study 2 / Exercise 3 Step 02 & 06. We are reviewing this and will provide an update once a resolution is in place.

**Comment by mstall1ngs** [comment]
Thanks again for providing insight on this. I was unable to replicate this as the Item number for L0025 displays the appropriate record on filtering. 

![image](https://github.com/MicrosoftLearning/MB-335-Dynamics-365-Supply-Chain-Management-Expert/assets/104943975/f19b8951-cc71-44b1-b5ec-cdd7932118e5)

I was also unable to replicate as I was able to choose Warehouse 13 for Site 1 - Finished Goods. 

![image](https://github.com/MicrosoftLearning/MB-335-Dynamics-365-Supply-Chain-Management-Expert/assets/104943975/d7e96d91-b3cd-40b8-9277-f5682cfb4dd7)

Closing as there are no updates currently required.
